src/sagemaker/tensorflow/estimator.py

```diff
@@ -275,13 +318,13 @@ def _prepare_init_params_from_job_description(cls, job_details, model_channel_na
                                                                                        model_channel_name)
 
         # Move some of the tensorflow specific init params from hyperparameters into the main init params.
-        for argument in ['checkpoint_path', 'training_steps', 'evaluation_steps']:
+        for argument in ['checkpoint_path', 'training_steps', 'evaluation_steps', 'model_dir']:
```

Comment:
I know this wasn't your change here, but you could make this list into a tuple as well